custom/languages/en_UK.json

```diff
@@ -128,7 +128,7 @@
     "admin\/default_avatar": "Default Avatar",
     "admin\/default_group": "Is the group the default group (for new users)?",
     "admin\/default_language": "Default Language",
-    "admin\/default_language_help": "Users will be able to choose from any installed languages.",
+    "admin\/default_language_help": "Users will be able to choose from any installed languages. <a target=\'_blank\' href=\'https://docs.namelessmc.com/en/home#translations\'>Read more »</a>",
```

Comment:
We not use html in language files, must make they as placeholders like we did with the other links like docLinkStart then cocLinkEnd

Reply:
Sorry, I was working with pr12 where `discord_invite_info` used to contain a hyperlink so I used that as reference.

That said, how would we handle i18n? For example, ultimately Spanish should use `docs.namelessmc.com/**es**/home#translations` instead of `docs.namelessmc.com/en/home#translations`, right?

Reply:
`docs.namelessmc.com/<page>` will redirect to `docs.namelessmc.com/<language>/<page>` based on the browser's language preference. Not sure how to handle sections (after `#`) though.

Reply:
That doesn't happen for me. Whether I set the browser language or the page language using the globe icon at top, going to `docs.namelessmc.com/<page>` in a new tab always redirects to English.

Reply:
I've made the requested change, regardless. Not sure if `installer/friendly_urls_info` should be updated to not use HTML as well, but that string is outside the scope of this PR.